tools/ports/regal.py

```diff
@@ -106,8 +106,6 @@ def create(final):
       '-I' + source_path_regal,
       '-I' + source_path_lookup3,
       '-I' + source_path_boost,
-      '-Wall',
-      '-Werror',
```

Comment:
`system_libs.get_base_cflags()` includes `-Werror` and `build_port` already adds this.

`-Wall` was removed here since it comes after the `-Wno-experimental -sMEMORY64=1` flags.. and so it undoes the `-Wno-experimental`